aiida/work/legacy/job_process.py

```diff
@@ -151,3 +151,9 @@ def _setup_db_record(self):
 
         if parent_calc:
             self._calc.add_link_from(parent_calc, "CALL", LinkType.CALL)
+
+        if self.raw_inputs:
```

Comment:
Or maybe a similar but related thing where some input was set to ``None`` and triggered a problem.